### ISSUE TITLE
Multlingual and custom pages: best way to document

### ISSUE THREAD

**Issue by brockfanning** [open-sdg/open-sdg#780]
Right now, when a country goes through the Quick Start and then wants to add a second language, they need to create second versions of all the custom pages (the About page, FAQ page, etc.). This is not obvious even though it's covered in the 4th item here: https://open-sdg.readthedocs.io/en/latest/translation/#adding-new-languages

We were brainstorming ways to make this easier for countries and discussed a few options:
1. Make the site starter begin with 2 languages instead of 1, so that it is clear how this would be done
2. Remove all the custom pages from the site starter, so that at least tests would not fail when a second language is added
3. Expand the documentation of this requirement
4. Figure out some way to automate the multilingual versions (perhaps with the automatic text "Translation not available yet" or similar)

**Comment by brockfanning** [comment]
I've made a couple of pull-requests that would implement option 1 above:
* https://github.com/open-sdg/open-sdg-data-starter/pull/37
* https://github.com/open-sdg/open-sdg-site-starter/pull/57

**Comment by brockfanning** [comment]
@LucyGwilliamAdmin Do you think we should go ahead with option 1 above? Or should we keep the starters at 1 language and try another option?

**Comment by brockfanning** [comment]
Documenting another option brought up by @jwestw: have the site starter start with all 6 of the official UN languages. The thinking is that it's easy to remove languages from the list of languages - and having extra pages being built is not so bad, since they are not linked anywhere.

**Comment by brockfanning** [comment]
@LucyGwilliamAdmin @jwestw One technical problem I just noticed - if a site has documents in a certain language, and then that language is removed from the site config `languages` list - the Jekyll build fails. I think we can fix that problem, but it would need to be done in the jekyll-open-sdg-plugins project. I've made an issue there: https://github.com/open-sdg/jekyll-open-sdg-plugins/issues/38

**Comment by brockfanning** [comment]
@LucyGwilliamAdmin @jwestw Update - that problem is resolved now.

**Comment by LucyGwilliamAdmin** [comment]
@brockfanning great! I'm still unsure of the best way to do this but personally leaning towards starting with 2 languages rather than all languages because

A. if countries want a second language, all they need to do is change the name of the language folders and the name of the language in the config files (as well as the content but would need to do that either way) 
B. If a country wants a third/fourth language, the existing second language folder serves as a good example of how to set up following languages
C. Removing the files for all languages will be very tedious - I know you mentioned that those who use git to make changes won't have much of a problem, but then it's also probably those with experience that wouldn't have a problem adding extra languages anyway.

**Comment by brockfanning** [comment]
@LucyGwilliamAdmin Those are good point. Another practical consideration is that the site builds take longer to complete if more languages are enabled - so with 6 languages enabled, the builds will take 3 times longer than with just 2. This wouldn't be a problem after the country removed the languages they didn't need, but those first few builds would be pretty slow.

**Comment by brockfanning** [comment]
@AnnCorp @jwestw I believe Lucy and I were leaning towards updating the starter repositories to start with two languages instead of 1. Do you think it's OK to go ahead with this plan?

**Comment by AnnCorp** [comment]
@brockfanning yes, sounds fine to me, based on my (limited) knowledge that its easier to disable a language if not needed than to add one and assuming by you and Lucy recommending this approach there wouldn't be big overheads or issues for single language implementations if do this?

**Comment by brockfanning** [comment]
Right, single-language implementations can easily remove the extra language, and then clean up the excess files at some point later, at their convenience.

**Comment by brockfanning** [comment]
Great, the starter repos are now set up for two languages. Closing this.
